test/lazy-loading.html

```diff
@@ -984,22 +985,6 @@
               expect(comboBox.value).to.equal('other value');
             });
 
-            it('should keep previous value if allow-custom-value is set', () => {
-              comboBox.allowCustomValue = true;
-              comboBox.open();
-              setInputValue('other value');
-              comboBox.inputElement.dispatchEvent(new CustomEvent('input'));
-              comboBox.close();
-              expect(comboBox.value).to.eql('other value');
-
-              comboBox.focus();
-              setInputValue('item 12');
```

Comment:
**note:** The test is incorrect and therefore removed. It's incorrect to set an input value without dispatching an input event. If we change it to dispatch the input event, it no longer works.